README.md

```diff
@@ -1,74 +1,53 @@
-- Introduce fine-grained location filtering based on the user’s current geolocation (e.g. search for all items in a 5km radius of their location)
-- Keep a record of the items each user has borrowed
-- Introduce some concept of renting (rather than borrowing) and include some payment procedures
+### Node & Express
+Our project used a Node server with Express for our backend. We used a simple session-based authentication scheme for user sign-in and authorization, and used Express middleware functions to easily add authentication/authorization checks to our API routes. We also wrote Express middleware for API error handling, and all of our routes send appropriate messages when an error occurs.
 
-## Project Task Requirements
-### Minimal requirements 
- - Should be able to create an account and enter basic profile information like name and password
- - Should be able to log in and out of the app
- - Should be able to view a list of items available for borrowing from other users
- - Should be able to create/delete/edit/view my owned items
- - Should be able to select an item from the list of items and borrow it on a first-come first-served basis
-### Standard requirements
- - Should be able to change the status of an item I own from “borrowed” to “returned”
- - Should be able to rate an item
- - Should be able to search for items
- - Should be able to filter items in the list (e.g. by rating, category)
- - Should be able to add basic location information to my profile, where the location is based on a set number of pre-filled neighbourhoods
-### Stretch requirements
- - Should be able to view a list of the items I’ve borrowed, and only be able to rate item’s I’ve borrowed
- - Should be able to send a request to borrow an item, view a list of requests for my items, and accept or decline a request
- - Should be able to filter items based on their distance from my current geolocation
+### NoSQL with MongoDB
+We used MongoDB to store our user, item, and request data, and Mongoose for our ODM. MongoDB was helpful as it allows for a flexible schema, which we had to update well into the development process to accomodate for the new requests feature. We wrote Mongoose middleware to do cleanup when we delete resources, for example automatically deleting uploaded images from our image storage when a user deletes an item.
 
-## Task Breakdown
- - Should be able to create an account and enter basic profile information like name and password
-   - Create a signup page
-   - Create a database table to store user information
-   - Set up a backend API route to register a new user
- - Should be able to view a list of items available for borrowing from other users
-   - Create a page that shows the user a list of items
-   - Create a database table to store items
-   - Set up a backend API route to list items
-   - Create a backend handler that queries the database to show all available items from other users
+### Release Engineering
```

Comment:
Trying to think about how to rephrase this so we can compare it to other deployment services and why this works well.